tiers/checks.go

```diff
@@ -73,26 +75,36 @@ func getDefaultChecks(nsType string) []innerObjectCheck {
 }
 
 func (a *basicTierChecks) GetExpectedRevisions(awaitility *wait.Awaitility) Revisions {
-	revisions := GetRevisions(awaitility, "basic", "code", "dev", "stage")
-	return revisions
+	return GetRevisions(awaitility.Host(), "basic")
+}
+
+func (a *basicTierChecks) GetClusterObjectChecks() []clusterObjectsCheck {
+	return []clusterObjectsCheck{
+		numberOfClusterResourceQuotas(1),
```

Comment:
Just checking the number of quotas is not enough. Let's also check the content. Similar to `limitRange()` check: https://github.com/codeready-toolchain/toolchain-e2e/blob/master/tiers/checks.go#L154

Also, different tiers will have different quotas. So the check should be parametrized.
See https://github.com/codeready-toolchain/host-operator/pull/177 and https://github.com/codeready-toolchain/toolchain-e2e/pull/97

Reply:
ok, makes sense